resources/views/cart/partials/top-bar.blade.php

```diff
@@ -1,10 +1,15 @@
+@php($checkoutSteps = config('rapidez.frontend.checkout_steps.' . config('rapidez.store_code')) ?: config('rapidez.frontend.checkout_steps.default'))
```

Comment:
Do we still needs this with: https://github.com/rapidez/core/blob/master/src/Http/Controllers/CheckoutController.php#L24, can't we use the variable from the controller?

Reply:
Neat, definitely would replace that here.

Funny story, turns out this file isn't even used anymore so I'm deleting it instead 🙂